docs/game_sdk/Networking.md

```diff
@@ -51,90 +51,135 @@ OnLateUpdate()
 
 ## OpenChannel
 
-Opens an unreliable channel to the given session id on the given channel number. Use this type of channel for loss-tolerant data, like player positioning in the world.
+Opens a channel to a user with their given peer on the given channel number.
+
+Unreliable channels—`reliable = false`—should be used for loss-tolerant data, like player positioning in the world. Reliable channels—`reliable = true`—should be used for data that _must_ get to the user, like loot drops!
 
 Returns `void`.
 
 ###### Parameters
 
-| name            | type   | description                     |
-| --------------- | ------ | ------------------------------- |
-| remoteSessionId | UInt64 | the session id to connect to    |
-| channel         | byte   | the channel on which to connect |
+| name      | type   | description                                          |
+| --------- | ------ | ---------------------------------------------------- |
+| peerId    | UInt64 | the peerId of the user to connect to                 |
+| channelId | byte   | the channel on which to connect                      |
+| reliable  | bool   | whether the channel should be unreliable or reliable |
 
 ###### Example
 
 ```cs
 // In reality, you'd fetch this from their lobby metadata
-var playerARemoteSessionId = 1234;
-networkManager.OpenChannel(playerARemoteSessionId, 0);
+var playerAPeerId = 1234;
+networkManager.OpenChannel(playerAPeerId, 0, false);
 ```
 
-## OpenReliableChannel
+## OpenPeer
 
-Opens a reliable channel to the given session id on the given channel number. Use this type of channel for data that _must_ get to the user, like loot drops!
+Opens a network connection to another Discord user.
 
 Returns `void`.
 
 ###### Parameters
 
-| name            | type   | description                     |
-| --------------- | ------ | ------------------------------- |
-| remoteSessionId | UInt64 | the session id to connect to    |
-| channel         | byte   | the channel on which to connect |
+| name   | type   | description                                  |
+| ------ | ------ | -------------------------------------------- |
+| peerId | UInt64 | the peerId of the user to connect to         |
+| route  | byte[] | the route the user is currently broadcasting |
 
 ###### Example
 
 ```cs
-// In reality, you'd fetch this from their lobby metadata
-var playerARemoteSessionId = 1234;
-networkManager.OpenReliableChannel(playerARemoteSessionId, 1);
+networkManager.OpenPeer(1, 2222);
```

Comment:
Change `2222` here to a variable name like `route_data`, since there isn't a literal you could put in this example that would make sense.

Reply:
Oh oops thought I fixed that. Good ol' filler data